src/autopredictor/select_model.py

```diff
@@ -0,0 +1,21 @@
+def select_model(df_output, model_name):
+    """
+    Selects and returns the scores of a specified model from a DataFrame of model scores.
+
+    This function searches a DataFrame, typically generated by the show_all() function,
+    for a model matching the name being provided. If found, it returns the corresponding row 
+    of scores. Otherwise, it issues a "Model Not Found""Model Not Found" message.
+
+    Parameters
+    ----------
+    df_output : DataFrame
```

Comment:
Let me know if you want me to change mine from Pandas DataFrame to DataFrame, or you want to change yours to Pandas DataFrame. Honestly, I feel like DataFrame is just fine, so I should change mine, but let me know.

Reply:
I believe both options are suitable for documentation. If we do change it to pandas dataframe I will also need to specify also Pandas Series for output. I am content with either choice.

Reply:
This was resolved by updating the fit function